packages/android_intent/example/lib/main.dart

```diff
@@ -162,6 +172,11 @@ class ExplicitIntentsWidget extends StatelessWidget {
                 child: const Text('Tap here to open link in Google Chrome.'),
                 onPressed: _openLinkInGoogleChrome,
               ),
+              RaisedButton(
+                child: const Text(
+                    'Tap here to open link in browser in a seperated window.'),
+                onPressed: _openLinkInDefaultBrowserInSeperatedWindow,
```

Comment:
I like a close-to-reality example like this one, however I would name it more accurate ("Start Activity in New Task"). What other ways could we use in the example to demonstrate the functionality?